packages/eslint-config-datacamp/index.js

```diff
@@ -50,8 +50,9 @@ module.exports = {
     'comma-dangle': 'off', // Defined by prettier
     'eslint-comments/no-unused-disable': 'error',
     'eslint-comments/no-unused-enable': 'error',
-    'filenames/match-exported': ['error', [null, 'camel'], null, true],
+    'filenames/match-exported': ['error', [null, 'camel', 'kebab'], null, true],
     'import/no-anonymous-default-export': ['error', { allowObject: true }],
+    'import/no-deprecated': 'warn',
```

Comment:
```suggestion
    'import/no-deprecated': 'error',
```

Should we only warn or should we error? Most apps / teams will ignore warnings, so it would only be for the developer seeing the eslintw arning then I guess. (I think in general it's already crossed out in your editor, if you set it up correctly)

I can live with this for now and we can always change later of course...

Reply:
Let's see how it works, and later we can change to 'error'.